cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1118,12 +1118,33 @@ function WaitFor_KubeletAndKubeProxyReady {
 }
 
 # Runs 'kubectl get nodes'.
-# TODO(pjh): run more verification commands.
+# Runs additional verification commands to ensure node succesfully joined cluster
+# and that it connects to the API Server.
 function Verify-WorkerServices {
-  Log-Output ("kubectl get nodes:`n" +
-              $(& "${env:NODE_DIR}\kubectl.exe" get nodes | Out-String))
+  $timeout = 12
+  $retries = 0
+  
+  Write-Host("Testing node connection to API server...")
```

Comment:
Let's use Log-Output?

Reply:
Yes! I was debating that in my mind, thank you